src/commands/kthid/kthid.handler.ts

```diff
@@ -0,0 +1,23 @@
+import { getKthIdByUserId } from "../../db/db";
+import { GuildChatInputCommandInteraction } from "../../shared/types/GuildChatInputCommandType";
+import { KthIdVariables } from "./kthid.variables";
+
+export const handleKthId = async (
+	interaction: GuildChatInputCommandInteraction
+): Promise<void> => {
+	const { options } = interaction;
+	const user = options
+		.getString(KthIdVariables.USER, true)
+		.replace(/[\D]/g, "");
+	const kthId = await getKthIdByUserId(user);
+	await interaction.deferReply({ ephemeral: true });
```

Comment:
`deferReply` säger till Discord "Jaja, jag lever. Så tro inte att boten är död om det tar lite lång tid.". Om man gör något som tar över en sekund kommer Discord att ge ett felmeddelande till användaren om att boten inte svarar så för säkerhetens skull kallar vi på `deferReply` i början av alla kommandon. Nu är databashämtningen väldigt snabb men i teori kanske det skulle finnas en situation då den tar lite tid så kan vara bra att ha kallat på `deferReply` innan `getKthIdByUserId`.
```suggestion
	await interaction.deferReply({ ephemeral: true });
	const kthId = await getKthIdByUserId(user);
```